### ISSUE TITLE
Use `{touchstone}` for benchmarking

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/workflows#8]
https://github.com/lorenzwalthert/touchstone

Probably relevant only after stable release when we want to focus on improving performance.

**Comment by etiennebacher** [comment]
I'm not convinced this is useful. It's nice to have good performance but it's not the selling point of `easystats`. I wouldn't even know what to benchmark given the number of functions and methods supported. @IndrajeetPatil feel free to reopen if you still find it relevant.
